providers/base/units/led/test-plan.pxu

```diff
@@ -5,8 +5,6 @@ _description:
  LED tests
  Notes: - led/power-blink-suspend and led/suspend are used later in the
           power management testplan just after a number of suspend tests.
-        - led/wireless is redundant given that we have led/wlan and
-          led/bluetooth already.
```

Comment:
Doesn't this say the opposite of what you did? `led/wireless` is not needed as we already have two other tests that cover it and more.

Reply:
Yes indeed, but in the end, the opposite was done: `led/wireless` was used in the test plans, and `led/wlan` and `led/bluetooth` were removed. It's a case of a comment that was never updated...